_layouts/home.html

```diff
@@ -37,6 +45,14 @@ <h1 style="border-bottom: 1px solid #ccc">
             <section id="mission">
                 {{ content }}
             </section>
+            <section id="minutes">
+                <h2>Telco Minutes</h2>
+                <ul class="inline">
+                {% for item in site.minutes reversed %}
```

Comment:
Probably should limit this to a handful of recent minutes and a "more..." link to the minutes page.  Would be a bit cluttered after we've had many meetings.

Reply:
👍 fixed in 7ccf60f